apps/web/src/utils/string.ts

```diff
@@ -0,0 +1,5 @@
+export function capitalizeFirstLetter(string) {
+  if (!string) return '';
+
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
```

Comment:
```suggestion
export function capitalizeFirstLetter(word: string) {
   return !word ? '' : string.charAt(0).toUpperCase() + string.slice(1);
}
```
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** We should ideally be using CSS for this when possible‏